website/docs/integrations/github-actions/atmos-terraform-plan.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Atmos Terraform Plan
+sidebar_position: 40
+sidebar_label: Terraform Plan
+---
+
+The Cloud Posse GitHub Action for "Atmos Terraform Plan" runs Terraform entirely from GitHub Action workflows.
+
+Given any component and stack in an Atmos supported infrastructure environment, [`github-action-atmos-terraform-plan`](https://github.com/cloudposse/github-action-atmos-terraform-plan) will run `atmos terraform plan`, generate a Terraform [planfile](https://developer.hashicorp.com/terraform/tutorials/automation/automate-terraform), store this planfile in an S3 Bucket with metadata in DynamodDB, and finally format the Terraform Plan result as part of a [GitHub Workflow Job Summary](https://github.blog/2022-05-09-supercharging-github-actions-with-job-summaries/).
+
+This action is intended to be used with [Atmos Terraform Apply](/integrations/github-actions/atmos-terraform-apply)
+
+## Features
+
+This GitHub Action incorporates superior GitOps support for Terraform by utilizing the capabilities of Atmos, enabling efficient management of large enterprise-scale environments.
+
+* **Implements Native GitOps** with Atmos and Terraform
+* **No hardcoded credentials.** Use GitHub OIDC to assume roles.
+* **Compatible with GitHub Cloud & Self-hosted Runners** for maximum flexibility. 
+* **Beautiful Job Summaries** don't clutter up pull requests with noisy GitHub comments
+* **100% Open Source with Permissive APACHE2 License** means you have no expensive subscriptions or long-term commitments.
+
+![Example Image](/img/github-actions/tf_plan.png)
+
+![Example Image Expanded](/img/github-actions/tf_plan_expanded.png)
+
+## Usage Example
+
+```yaml
+name: "atmos-terraform-plan"
+
+on:
+  workflow_dispatch:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+    branches:
+      - main
+
+# These permissions are required for GitHub to assume roles in AWS
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  plan:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Plan Atmos Component
+        uses: cloudposse/github-action-atmos-terraform-plan@v1
+        with:
+          component: "foobar"
+          stack: "plat-ue2-sandbox"
+          component-path: "components/terraform/s3-bucket"
+          terraform-plan-role: "arn:aws:iam::111111111111:role/acme-core-gbl-identity-gitops"
+          terraform-state-bucket: "acme-core-ue2-auto-gitops"
+          terraform-state-role: "arn:aws:iam::999999999999:role/acme-core-ue2-auto-gitops-gha"
+          terraform-state-table: "acme-core-ue2-auto-gitops"
+          aws-region: "us-east-2"
+
+```
+
```

Comment:
```suggestion
```

Please note that in practice, we recommend combining this action with the [`affected-stacks`](/integrations/github-actions/affected-stacks) GitHub Action inside a matrix to plan all affected stacks in parallel.
```